### ISSUE TITLE
case when digitalme is not installed results in development breakage

### ISSUE THREAD

**Issue by delandtj** [threefoldtech/jumpscale_core#94]
when digitalme is not installed (which is new so will not be for most people), the test "if not j.application.schemas" will break (throw an exception) because that property does not exist.

therefore the entire application will break and cannot start up.. unless digitalme is installed.

**Comment by rkhamis** [comment]
duplicate of https://github.com/threefoldtech/jumpscale_core/issues/78
